linear_operator/operators/_linear_operator.py

```diff
@@ -1964,6 +1964,14 @@ def requires_grad_(self, val: bool) -> LinearOperator:
         self._set_requires_grad(val)
         return self
 
+    def reshape(self, *sizes: Union[torch.Size, Tuple[int, ...]]) -> LinearOperator:
+        """
+        Alias for expand
+        """
+        # TODO: Do we really want to do this? I guess the difference betwen reshape and
+        # expand is kind of unclear for LinearOperator.
+        return self.expand(*sizes)
```

Comment:
TODO: Raise an informative warning if this tries to make modify the last two dimensions (that we have to think about more carefully when supporting non-square tensors more broadly).

Reply:
Currently, "expand" errors out when you expand the last two dimensions, so I think this PR is okay as is without a warning.